### DISCUSSION TITLE
c# source code convert to pascal source code?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2472] [Ideas]
very good this

**Comment by ibond84** [comment]
yes. C# <-> Pascal would be very helpfull

**Reply by ghost** [reply]
I developed a CSharp to Object Pascal converter, it's attache
[CSharpToObjectPascal.zip](https://github.com/pascalabcnet/pascalabcnet/files/7315648/CSharpToObjectPascal.zip)
d
